packages/hdwallet-native/src/bitcoin.ts

```diff
@@ -1,5 +1,5 @@
-import { ECPairInterface } from "bitcoinjs-lib";
-import * as bitcoin from "bitcoinjs-lib";
+import { ECPairInterface } from "@bithighlander/bitcoin-cash-js-lib";
```

Comment:
Would just remove the destructure and update to `bitcoin.ECPairInterface` where used. Only one import per package that way.